core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowAkkaPlugin.scala

```diff
@@ -24,32 +24,19 @@ import com.typesafe.sbt.packager.Keys._
 import spray.json._
 import cloudflow.sbt.CloudflowKeys._
 import cloudflow.blueprint.StreamletDescriptorFormat._
+import CloudflowBasePlugin._
 
-object CloudflowAkkaPlugin extends CloudflowBasePlugin {
+object CloudflowAkkaPlugin extends AutoPlugin {
+
+  override def requires = CloudflowBasePlugin
```

Comment:
ahh, so Autoplugin can "require" the behavior and settings of a another plugin (?)

Reply:
yes! That's more idiomatic way of composing plugins instead of using inheritance